tests/integration/sniffer/capturer.py

```
@@ -0,0 +1,858 @@
+    src_port,
+    dst_port,
+    req_headers=None,
+    path=None,
+    payload=None,
+    request=None,
+    response=None,
+    resp_headers=None,
+    resp_status=None,
+    resp_reason=None,
+    method=None,
+    additional_data={},
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)